### DISCUSSION TITLE
반성문3

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#647] [6. 반성문]
안녕하세요... 반성문 챌린지 죄송합니다...

저번주에는 쉬느라 글을 못썼습니다. 이번주는... 이번주도 아무래도 글쓰기는 무리일 것 같네요 🥲 어김없이 잘 이겨내보겠습니다.

최근에 깨달은 건데 저는 생각이 아주 많은 편이라서 여기저기 날아다니고, 휘발되는 생각들을 잡아다가 글로 옮겨적고 천천히 읽어보는 것이 정리가 많이 되고 스스로에게 도움이 많이 되더라구요...! 인턴 생활의 유종의 미를 거두고, 한번 전체적으로 회고를 하는 글을 작성해서 들고 오겠습니다.

**Comment by 02ggang9** [comment]
보름 화이팅입니다~

이번에도 잘 이겨낼거라 믿어용 ㅎ.ㅎ

**Reply by shkisme** [reply]
고맙숩니다!!! 팟팅해보겠습니다!!! 👏

**Comment by pykido** [comment]
화이팅입니다~!~!#!@#~!@!~@!~@~!@!~@~!@!~~!!~!@~!@!~@!@~@!~@!~@!
